docs/community/KPI-options.md

```diff
@@ -0,0 +1,41 @@
+---
+title: KPI Options
+sidebar_label: KPI Options
+---
+## KPI Option Overview
+
+Community options, growth options or performance options - the naming is not the point, more that these options are designed to create aligned incentives with your community to grow. We have decided to call them *Key Performance Indicator options* or *KPI options*. 
+
+### What are these options? 
+
+Using UMA’s Expiring Multi Party (EMP) contract template, any project can create an incentive mechanism which offers rewards to their targeted community in exchange for the community’s support in helping them achieve their Key Performance Metrics. 
+
+At a high-level, KPI options are synthetic ERC20 tokens that can be redeemed , but only if the project hits it’s predetermined KPI targets before expiry. These targets are up to each project and could be TVL, number of contracts deployed, number of transactions, etc.
+
+Every community member airdropped these KPI options, becoming vested in helping the project succeed on its Key Metrics, as the token grows more valuable as the target approaches.  This provides powerful incentive alignment between the protocol and its community. 
```

Comment:
If we move to @mhairimca's introduction, a lot of this content will become redundant. I think this section could get more granular and be a mix of What a KPI option is, and how the contract works.
- Expiring contract
- Sponsor mints erc20 synths backed by their protocol token, and airdrops them to protocol participants
- Price of the synth is based upon KPI thresholds
- At expiry, the synth becomes worth XYZ from KPI calculation

Just spitballing, but I think getting into the weeds on how the contract works would be beneficial and avoid redundant content.

Reply:
I was thinking of adding an example of how this works which would cover this section